src/main/kotlin/com/franzi/todochallenge/controller/TaskRepository.kt

```diff
@@ -0,0 +1,51 @@
+package com.franzi.todochallenge.repository
+
+
+import com.franzi.todochallenge.controller.Task
+import org.springframework.stereotype.Repository
+import java.lang.RuntimeException
+import java.time.Instant
+import java.util.*
+
+@Repository
+class TaskRepository {
+    private val tasksMap = mutableMapOf<UUID, Task>()
+
+    fun findAll(): List<Task> {
+        return tasksMap.toList().map { it.second }
+    }
+
+    fun findById(id: UUID): Task {
+        val task = tasksMap[id] //return tasksMap.get(id)
+        if (task == null) {
+            throw RuntimeException("Task with id: $id not found") // TODO: handle it and give back a proper http response to client
+        }
+
+        return task
```

Comment:
I think this would work
```suggestion
        return tasksMap[id] ?: throw RuntimeException("Task with id: $id not found") // TODO: handle it and give back a proper http response to client
```

Reply:
ok thats way shorter :) will change